docs/source/sources.rst

```diff
@@ -83,3 +83,15 @@ Utility Methods (:py:mod:`mira.sources.util`)
 .. automodule:: mira.sources.util
     :members:
     :show-inheritance:
+
```

Comment:
We would want to add a block here for the module i.e., `mira.source.system_dynamics.pysd`.